apps/graphql/src/types/output/BookingInterface.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import {
+  GraphQLInterfaceType,
+  GraphQLString,
+  GraphQLEnumType,
+  GraphQLInt,
+} from 'graphql';
+import globalID from '@kiwicom/graphql-global-id';
+
+import GraphQLRouteStop from './RouteStop';
+import type { Booking } from '../../dataloaders/BookingsLoader';
+import GraphQLBookingType from './BookingType';
+
+export const commonFields = {
+  id: globalID(({ bid }) => bid),
+  status: {
+    type: GraphQLString,
+  },
+  arrival: {
+    type: GraphQLRouteStop,
+  },
+  departure: {
+    type: GraphQLRouteStop,
+  },
+  destinationImageUrl: {
+    type: GraphQLString,
+    args: {
+      dimensions: {
+        type: new GraphQLEnumType({
```

Comment:
Should extract this GraphQL type to its own file?

Reply:
It is quite specific for just this one field, but if you think it is better, I can extract it.

Reply:
Do you think we won't need to re-use it? If that's the case, then I'm fine with leaving this at it is.

Reply:
> won't need to re-use it

definitely not